x-pack/filebeat/input/entityanalytics/provider/okta/statestore.go

```diff
@@ -188,9 +186,7 @@ func (s *stateStore) close(commit bool) (err error) {
 		}
 		rollbackErr := s.tx.Rollback()
 		if rollbackErr == nil {
-			// FIXME: Use fmt.Errorf("multiple errors during statestore close: %w", errors.Join(err, rollbackErr))
-			// when go1.20 is supported.
-			err = sderr.WrapAll([]error{err, rollbackErr}, "multiple errors during statestore close")
+			err = fmt.Errorf("multiple errors during statestore close: %w", errors.Join(err, rollbackErr))
```

Comment:
Do we need the to include `rollbackErr` in the`errors.Join` here?  We know `rollbackErr == nil` so won't it just get discarded?

Reply:
I can just be done non-conditionally. I don't know what I was thinking.
```
		err = fmt.Errorf("multiple errors during statestore close: %w", errors.Join(err, s.tx.Rollback()))
```